Installation/Action/hp.HandleSendEmailAction.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$id": "https://hp.nl/action/hp.HandleSendEmailAction.action.json",
+  "$schema": "https://docs.commongateway.nl/schemas/Action.schema.json",
+  "version": "0.0.4",
```

Comment:
Could be 0.0.1

Reply:
This is a action that was already in the repository. I only changed the location